### PR TITLE
Add QGB docs

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

- [x] add intro from https://github.com/celestiaorg/docs/pull/279
- [x] update go version in README.md https://github.com/celestiaorg/orchestrator-relayer/pull/312
- [x] change title of `Relayer` to `IBC Relayer`
- [x] set up automation to pull in outside markdown
  - ~~the current set up uses a react component to render the markdown, this may not be the best way, because headings do not have anchor links~~
- [x] add Validators category
- [x] add `Validator node`, `QGB orchestrator`, and `QGB keys subcommand` to Validators category
- [x] change validator section to this (the trick here is there is no native support in docusaurus for nested items, only category > category > category, which would be messy):
- Validators
  - Validator node
    - QGB Orchestrator
      - Keys subcommand
- [x] change the Celestiums category to:
- Celestiums (QGB)
  - QGB Relayer
    - Deploy the QGB contract
- [x] add `Celestiums (QGB)` category
- [x] add `QGB Relayer` and `Deploy the QGB contract` pages to `Celestiums (QGB)` category
- [x] ~~make PRs to~~ resolve issues with table formatting on source markdown
- [x] fix linting and standardize titles https://github.com/celestiaorg/orchestrator-relayer/pull/295 after we decide on structure. I am leaning toward V1 but V2 also works

### Final version of menu

scrot 2023-04-28 at 11 54 35 AM

<details>
<summary>Draft menu previews</summary>
<br/>

### V3 (I think this looks messy)

scrot 2023-04-18 at 6 27 39 PM

### V2 (current in preview)

scrot 2023-04-18 at 10 16 20 PM

### V1
scrot 2023-04-18 at 5 41 10 PM
</details>

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/qgb-deploy.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Deploy
```
Comment (SweeXordious) [inline docs/nodes/qgb-deploy.md:1]: we're no longer importing the markdown? I see down there that it's being imported from the orchestrator relayer repo, or I am missing something

**Review by SweeXordious — COMMENTED** [review]
The layout looks good to me :+1: :+1:

File: docs/nodes/qgb-deploy.md (hunk 1 above)
Comment (SweeXordious) [reply docs/nodes/qgb-deploy.md:1]: fancyyy

**Comment by jcstein** [comment]
Add intro from https://github.com/celestiaorg/docs/pull/279

File: docs/nodes/qgb-deploy.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  -n, --starting-nonce string   Specify the nonce to start the QGB contract from. "earliest": for genesis, "latest": for latest valset nonce, "nonce": for the latest valset before the provided nonce, provided nonce included. (default "latest")
+
+Use "qgb deploy [command] --help" for more information about a command.
+```
+
+## How to run
+
+### Install the QGB binary
+
+Make sure to have the QGB binary installed. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/README.md) for more details.
```
Comment (YazzyYaz) [inline docs/nodes/qgb-deploy.md:37]: let's make a task for a future PR to show how to install the binary inside the docs so folks don't leave the docs. not urgent for now.

File: docs/nodes/qgb-orchestrator.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+I[2023-04-26|00:04:08.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+I[2023-04-26|00:04:18.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+I[2023-04-26|00:04:28.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+```
+
+## How to run
+
+### Install the QGB binary
+
+Make sure to have the QGB binary installed. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/README.md) for more details.
```
Comment (YazzyYaz) [inline docs/nodes/qgb-orchestrator.md:32]: same here, re: show folks in the future how to install the binary within the docs.

**Review by YazzyYaz — APPROVED** [review]

File: docs/nodes/qgb-relayer.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+I[2023-04-26|00:04:08.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+I[2023-04-26|00:04:18.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+I[2023-04-26|00:04:28.175] waiting for routing table to populate        targetnumberofpeers=1 currentcount=0
+```
+
+## How to run
+
+### Install the QGB binary
+
+Make sure to have the QGB binary installed. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/README.md) for more details.
```
Comment (jcstein) [inline docs/nodes/qgb-relayer.md:36]: also here

**Pull request merged**